File: backend/src/main/java/com/example/backend/dto/CustomizationDTO.java
package com.example.backend.dto;

import java.math.BigDecimal;
import java.util.UUID;

public class CustomizationDTO {
    private UUID customizationId;
    private String name;
    private BigDecimal price;
    private UUID restaurantId;

    public UUID getCustomizationId() { return customizationId; }
    public void setCustomizationId(UUID customizationId) { this.customizationId = customizationId; }

    public String getName() { return name; }
    public void setName(String name) { this.name = name; }

    public BigDecimal getPrice() { return price; }
    public void setPrice(BigDecimal price) { this.price = price; }

    public UUID getRestaurantId() { return restaurantId; }
    public void setRestaurantId(UUID restaurantId) { this.restaurantId = restaurantId; }
}

File: frontend/src/dto/customization.dto.ts
export interface CustomizationDTO  {
  customizationId: string;
  name: string;
  price: number;
  restaurantId: string;
};

export type CustomizationCreateRequest = {
  name: string;
  price: string;
  restaurantId: string;
  categoryId?: string;
  menuItemId?: string;
};